griptape/engines/query/vector_query_engine.py

```diff
@@ -28,28 +33,26 @@ def query(
         artifacts = [
             a for a in [BaseArtifact.from_json(r.meta["artifact"]) for r in result] if isinstance(a, TextArtifact)
         ]
-
-        prefix_list = [
-            "You are a helpful assistant who can answer questions by searching through text segments.",
-            "Always be truthful. Don't make up facts.",
-            "Use the below list of text segments and optional metadata to answer the subsequent question.",
-            'If the answer cannot be found in the segments, say "I could not find an answer."'
-        ]
-
-        metadata = f"Metadata: {metadata if metadata else 'no metadata available'}"
-        prefix = f"{' '.join(prefix_list)}\n\n{metadata}"
-        question = f"\n\nQuestion: {query}"
-        answer = "\n\nAnswer: "
+        prefix = []
 
         for artifact in artifacts:
             next_segment = f'\n\nText segment:\n"""\n{artifact.value}\n"""'
```

Comment:
We should be able to tuck "Text segment" into a template as well and just use a for loop there:

```
{% for segment in text_segments %}
...
{% endfor %}
```